spec/system/planning_applications/validating_spec.rb

```diff
@@ -55,6 +55,19 @@
     expect(page).to have_text(assessor.name)
     expect(page).to have_text(Audit.last.created_at.strftime("%d-%m-%Y %H:%M"))
   end
+
+  it "allows document edit, archive and upload after invalidation" do
+    click_link planning_application.reference
+    click_link "Validate documents"
+
+    choose "No"
+
+    click_button "Save"
+
+    expect(page).to have_content("Application has been invalidated")
+
+    planning_application.reload
+  end
```

Comment:
not sure this test does what the description says? it just seems to be invalidating a planning application, not "allowing documents to be edited, archived and uploaded"?